analysis/merge.Rmd

```diff
@@ -0,0 +1,271 @@
+---
+title: "Integrating Conflict Data"
+output:
+  workflowr::wflow_html:
+    toc: false
+editor_options:
+  chunk_output_type: console
+---
+
+This page shows how to merge Geo-PKO data with conflict data and visualise the results. The examples used here are Uppsala University's ViEWS project, which forecasts conflict risk, and the Uppsala Conflict Data Programme (UCDP), one of the world's leading sources of data on armed conflict. Merging these datasets can provide insights into the links between conflict risk and peacekeeping deployments, and help policymakers make effective peacekeeping decisions where the risk of conflict is high.
+
+## Setting up
+
+Load packages.
+```{r, warning=FALSE, message=FALSE}
+library(sp)
+library(tidyr)
+library(dplyr)
+library(geojsonio)
+library(broom)
+library(rgdal)
+library(tidyverse)
+library(ggplot2)
+library(leaflet)
+library(sf)
+library(spdep)
+library(maptools)
+library(plyr)
+library(rjson)
+library(RJSONIO)
+library(rmapshaper)
+library(htmltools)
+library(htmlwidgets)
+```
+
+## Geo-PKO x ViEWS
+First, we import the datasets. We're  using two sets of data from ViEWS. Both forecast the risk of state-based conflict, non-state conflict, and one-sided violence over the next 36 months in Africa. One is at country level, and the other is at PRIO-grid level, an innovative geospatial unit from the Peace Research Institute Oslo that divides the world into roughly 100km x 100km squares. We're using a version of Geo-PKO data that has been coded to the ViEWS format and includes numerical month IDs and PRIO-GRID IDs. And lastly, we're also importing a version of the Geo-PKO data at the country level.
+
+```{r}
+ geopkoviews <- read.csv("geopko_pgm.csv")
```

Comment:
Please add these files to the repo in the "data" folder so I can build it on my computer!

Reply:
For the geopkoviews, it could be useful to offer a preview of what this dataset looks like and how it differs from the regular geopko -- after you explain the merge.